mscore/excerptsdialog.cpp

```diff
@@ -605,7 +605,11 @@ void ExcerptsDialog::createExcerptClicked(QListWidgetItem* cur)
     title->setEnabled(false);
 }
 
-Excerpt* ExcerptsDialog::prepareMovementExcerpt(Excerpt* masterExcerpt, MasterScore* score)
+//---------------------------------------------------------
+//   prepareMovementExcerpt
+//---------------------------------------------------------
+
+Excerpt* Album::prepareMovementExcerpt(Excerpt* masterExcerpt, MasterScore* score)
```

Comment:
static in header but not here

Reply:
Been that way before too though, just in a different header

Reply:
This is in accordance with the rules of C++. 'static' can only be specified inside the class definition.

Reply:
I'm probably too much of a C person...